Калькулятор/calc.js

```diff
@@ -0,0 +1,71 @@
+let a = '';
+let b = '';
```

Comment:
all your variables are declared in global `window` scope. try wrapping up them in self invoking function (details here https://www.w3schools.com/js/js_function_definition.asp#:~:text=Self%2DInvoking%20Functions&text=A%20self%2Dinvoking%20expression%20is,self%2Dinvoke%20a%20function%20declaration.)